community/connections/AnyRail/index.shtml

```diff
@@ -131,9 +131,18 @@
       </ol>
 
 
-
     <h2>Additional notes for using AnyRail's export files</h2>
 
+      <h3>Migration script</h3>
+      The
+      <a href="https://jmri.org/jython/anyrail/AnyRailBuildBlockSensorList.py">AnyRailBuildBlockSensorList.py</a>
+      script may be useful when doing large migrations with block sensors.
+      See the associated
+      <a href="./AnyRailBuildBlockSensorList.pdf">PDF instructions</a>
+      and
+      <a href="https://jmri.org/jython/anyrail/README.md">README.md</a>
+      files for more information.</li>
```

Comment:
There is a `</li>` on the end of the last line that doesn't seem to have a corresponding `<li>`.